environment.yml

```diff
@@ -33,6 +33,7 @@ dependencies:
   - numpydoc>=0.8
   - packaging
   - pydata-sphinx-theme
+  - pyyaml
```

Comment:
Note the new doc-build dependency.  However, this is hardly a heavy dependency.

Reply:
Maybe?  All our config files for GitHub etc are yaml.  We have `pyproject.toml`, but that is it... 
The resulting toml file looks like:

```toml
skip_subdirs = [ "users/prev_whats_new/*", "api/*", "gallery/*", "tutorials/*", "plot_types/*", "devel/*",]
comment = "For use with make html-skip-subdirs"
```

The yaml file:

```yaml
comment: For use with make html-skip-subdirs
skip_subdirs:
- users/prev_whats_new/*
- api/*
- gallery/*
- tutorials/*
- plot_types/*
- devel/*
```
The latter seems easier to comment etc.  But I don't feel strongly about it.

Reply:
You can split the toml list over multiple lines too (basically as in python), and commenting works just as well.

Reply:
Sure, but not with `toml.dump` so far as I can tell, which is how I make the default file.  Of course we could write by hand, but writing our own dumper seems like a PITA for this use case.  Is there some reason you prefer toml other than being part of the core library (at some point)?

Reply:
> Is there some reason you prefer toml other than being part of the core library (at some point)?

No. I don't feel strongly about it, in fact.